openlibrary/templates/account/books.html

```diff
@@ -64,9 +64,9 @@ <h3>
       <div class="mybooks-menu">
         <ul class="preset-lists">
           <li class="subsection">Reading Log</li>
-          <li><a href="$urlbase/currently-reading" $('class=selected' if key == 'currently-reading' else '')>Currently Reading ($(reading_log_counts['currently-reading']))</a></li>
-          <li><a href="$urlbase/want-to-read" $('class=selected' if key == 'want-to-read' else '')>Want to Read ($(reading_log_counts['want-to-read']))</a></li>
-          <li><a href="$urlbase/already-read" $('class=selected' if key == 'already-read' else '')>Already Read ($(reading_log_counts['already-read']))</a></li>
+          <li><a href="$urlbase/currently-reading" $('class=selected' if key == 'currently-reading' else '')>Currently Reading</a></li>
+          <li><a href="$urlbase/want-to-read" $('class=selected' if key == 'want-to-read' else '')>Want to Read</a></li>
+          <li><a href="$urlbase/already-read" $('class=selected' if key == 'already-read' else '')>Already Read</a></li>
```

Comment:
Let's revert this change; as a user of this feature on GoodReads, I would rather have the numbers be off when I update the page than not have them there at all.

Reply:
Done